pkg/test/env/istio.go

```diff
@@ -50,6 +50,10 @@ var (
 	// nolint: golint, revive, stylecheck
 	PULL_POLICY Variable = "PULL_POLICY"
 
+	// TEST_ECHO_IMAGE is the image to use when deploying echo services.
+	// nolint: golint, revive, stylecheck
+	TEST_ECHO_IMAGE Variable = "TEST_ECHO_IMAGE"
```

Comment:
nit: just call it `ECHO_IMAGE`. Everything here is for `test` :)

Reply:
Done!